### ISSUE TITLE
1.15.2 Development Bugs

### ISSUE THREAD

**Issue by MasterCubo** [sp614x/optifine#3365]
I'm not sure if I'm supposed to put it here or not, but in the unstable 1.15.2 release, when I set my Render Distance over 20 chunks, no chunks load at all. This could be a computer bug or a bug with Minecraft, but I thought I'd include it.

**Comment by Aternias** [comment]
I can confirm. I loaded onto a server with prerendered chunks and trying to load new chunks fails. The only thing that loads are entities, no blocks.

**Comment by MasterCubo** [comment]
Yes, I forgot to add the entities, but thats what I got.

**Comment by BoeingLikesToDab** [comment]
I have this issue as well but it doesn't matter if it's on a server or a local world, and my friends have this too, they don't render and the character gets stuck.

**Comment by TJArguedscarab** [comment]
Yeah I am having the same issues if players have the chunks loaded already than you should be good to go on servers. But in single player worlds since its the first time you load the chunk not all will load since it is a bug in the OptiFine mod and there is going to be bugs since we are not using the release version yet. When the new version come out maybe it will be fixed but not now

**Comment by i-spy** [comment]
having the exact same issue

**Comment by realjackasterisk** [comment]
Successfully reproduced by maxing render distance. Had to travel a bit after changing it, but it's there. Fixed by lowering the render distance to an extremely low value (less than 5 or so)

**Comment by Zxurian** [comment]
running `Optifine 1.15.2_HD_U_G1_pre18`
arrived here after googling the same issue I had. Running forge with Optifine as a loaded mod. Tried render distances 24/28/32/36/40/44/48, all of them wind up with random interspersed chunks just not loading with the rest of the world. Sometimes 1 or 2, sometimes an entire section. Screenshot attached. After moving Optifine out of loaded mods, problem didn't appear anymore.

![image](https://user-images.githubusercontent.com/573371/82642434-eb892c00-9bdb-11ea-9d3c-0aa84c51b7ad.png)

**Comment by realjackasterisk** [comment]
I'm gonna go ahead and remove the "multiplayer" label since this also occurs in singleplayer.

**Comment by realjackasterisk** [comment]
More details at #3752

**Comment by fun840** [comment]
I'm having the same issue, though I do have journeymap installed. I'm going to try it without journeymap and see if it fixes.

**Comment by fun840** [comment]
Ok, removing journeymap fixed it. Also if you're having this issue but don't want to remove journeymap, pressing f3+a will reload chunks.

**Comment by sp614x** [comment]
Relevant #3873.

**Comment by AlternateTheory** [comment]
I created issue [1.15.2 Pre17] Chunks not rendering #3962 that was linked to this issue.
Up until recently every pre release I tried still had the issue.

However I have tried OptiFine 1.16.2 HD U G3 pre6... and do not seem to have the issue anymore :)
I didn't do extensive testing, but I can now create a world on my local machine in creative and fly away from the origin and the chunks are rendering and showing properly... something that consistently was not working previously.

**Comment by sp614x** [comment]
This was fixed in 1.16.2 pre6.
The fix will be ported to 1.15.x when it is updated.

**Comment by Quizer9O8** [comment]
Has there been any progress in porting this feature over to 1.15.2 since 1.16.x for Forge isn't fully stable for some mods to start porting?
